lib/components/user/monitored-trip/trip-notifications-pane.js

```diff
@@ -44,64 +102,66 @@ class TripNotificationsPane extends Component {
     } else {
       notificationSettingsContent = (
         <>
-          <FormGroup>
-            <ControlLabel id='is-active-label'>Would you like to receive notifications about this trip?</ControlLabel>
-            <div role='group' aria-labelledby='is-active-label'>
-              <Radio
-                checked={monitoredTrip.isActive}
-                name='isActive'
-                onChange={this._handleIsActiveChange}
-                value
+          <h4>Notify me via {notificationChannelLabels[notificationChannel]} when:</h4>
+          <SettingsListWithAlign>
```

Comment:
After seeing this ini action, let's make all these settings flush with the left edge (i.e., no indentation). Same goes for the below item in Advanced Settings.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/323/commits/4801a00160dbc1d75bb375d39d2cbb0bb4b4c34e.